lib/cb/models/implementations/saved_search.rb

```diff
@@ -36,18 +36,18 @@ def initialize(args = {})
         @search_parameters          = SearchParameters.new(args['SavedSearchParameters'] || {})
       end
 
-      def create_to_xml
-        <<-eos
-          <Request>
-            <HostSite>#{host_site}</HostSite>
-            <Cobrand>#{cobrand}</Cobrand>
-            <SearchName>#{search_name}</SearchName>
-            #{search_parameters.to_xml}
-            <IsDailyEmail>#{is_daily_email.to_s.upcase}</IsDailyEmail>
-            <ExternalUserID>#{external_user_id}</ExternalUserID>
-            <DeveloperKey>#{Cb.configuration.dev_key}</DeveloperKey>
-          </Request>
-        eos
+      def create_to_json
+        hash = {
+          'SiteID' => site_id,
+          'Cobrand' => cobrand,
+          'EmailDeliveryDay' => email_delivery_day,
```

Comment:
Looks like the new version omits IsDailyEmail entirely while keeping EmailDeliveryDay. Line 49 now upcases EmailDeliveryDay if is_daily_email is non-nil. Is it ok that there's no case where IsDailyEmail is included in this request json?